README.md

```diff
@@ -1,2 +1,144 @@
+| `health_check_path`                |     `/`         | The destination for the health check request                                     |    No    |
+| `health_check_timeout`             |     `10`        | The amount of time to wait in seconds before failing a health check request      |    No    |
+| `health_check_healthy_threshold`   |     `2`         | The number of consecutive health checks successes required before healthy        |    No    |
+| `health_check_unhealthy_threshold` |     `2`         | The number of consecutive health check failures required before unhealthy        |    No    |
+| `health_check_interval`            |     `15`        | The duration in seconds in between health checks                                 |    No    |
+| `health_check_matcher`             |   `200-399`     | The HTTP response codes to indicate a healthy check                              |    No    |
+| `attributes`                       |     `[]`        | Additional attributes (e.g. `1`)                                                 |    No    |
+| `tags`                             |     `{}`        | Additional tags  (e.g. `map("BusinessUnit","XYZ")`                               |    No    |
+| `delimiter`                        |     `-`         | Delimiter to be used between `namespace`, `stage`, `name` and `attributes`       |    No    |
+
+
+
+## Outputs
+
+| Name                            | Description                                                     |
+|:--------------------------------|:----------------------------------------------------------------|
+| `alb_arn`                       | The ARN of the ALB                                              |
+| `alb_arn_suffix`                | The ARN suffix of the ALB                                       |
+| `alb_dns_name`                  | DNS name of ALB                                                 |
+| `alb_zone_id`                   | The ID of the zone which ALB is provisioned                     |
+| `security_group_id`             | The security group ID of the ALB                                |
+| `default_target_group_arn`      | The default target group ARN                                    |
+| `http_listener_arn`             | The ARN of the HTTP listener                                    |
+| `https_listener_arn`            | The ARN of the HTTPS listener                                   |
+| `listener_arns`                 | A list of all the listener ARNs                                 |
+| `access_logs_bucket_id`         | The S3 bucket ID for access logs                                |
+
+## Help
+
+**Got a question?**
+
+File a GitHub [issue](https://github.com/cloudposse/terraform-aws-alb/issues), send us an [email](mailto:hello@cloudposse.com) or reach out to us on [Gitter](https://gitter.im/cloudposse/).
```

Comment:
I think we are moving to this format:

Review the [docs](https://docs.cloudposse.com/), ask a [question](https://docs.cloudposse.com/discuss-new), file a GitHub [issue](https://github.com/cloudposse/terraform-aws-ssm-iam-role/issues), send us an [email](mailto:hello@cloudposse.com) or reach out to us on [Slack](https://slack.cloudposse.com).